weblogic-azure-aks/src/main/arm/scripts/createVMAndBuildImage.sh

```diff
@@ -6,16 +6,9 @@
 
 echo "Script  ${0} starts"
 
-# read <azureACRPassword> and <ocrSSOPSW> from stdin
+# read <acrPassword> from stdin
```

Comment:
To avoid using `export` variable to share acrPassword between scripts, keep using stdin here.